daemon/gluster-blockd.c

```diff
@@ -458,6 +458,14 @@ gbDependenciesVersionCheck(void)
   }
   GB_FREE(out);
 
+  out = gbRunnerGetOutput(CONFIGSHELL_VERSION);
+  if (!out[0]) {
+    LOG("mgmt", GB_LOG_INFO, "starting with configshell version < 1.1.25");
```

Comment:
@lxbsz may be you can print 'out' variable here.

Reply:
But it won't make sense here because it it null.

Reply:
oops! my bad, read the code wrong.